bin/document_slug_updater

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env ruby
+
+require File.expand_path("../../config/environment", __FILE__)
+require "./app/lib/specialist_publisher.rb"
+
+def strip_leading_slash(path)
+  path.sub(%r{^/}, "")
```

Comment:
It shouldn't make a difference here, but you probably mean `\A` to match the start of the input rather than `^` which matches the start of any line